.travis.yml

```diff
@@ -66,6 +66,22 @@ matrix:
        - sphinx-intl build --locale-dir=docs/locale/
        - sphinx-build -b html -D language=es -n docs docs/_build/html
 
+    # Memcache enabled
+    - php: 5.6
+      services:
+       - memcached
+       - mysql
+      env: VARIA=true
+      before_install:
+        - phpenv config-rm xdebug.ini
+        - phpenv config-add ./.scripts/travis/with-memcached.ini
+      install:
+       - composer travis:install-with-mysql
+      script:
+       - php -f ./.scripts/is_memcached_enabled.php
+       - ./vendor/bin/phpunit
+       - php -f ./engine/tests/suite.php
```

Comment:
Make a 2nd config file with memcache settings. Pass in a flag to tests/suite.php. If set, we load it by setting`$CONFIG->Config_file`. Problem is `Application::create()` is private. We need to call it, set the $CONFIG value, *then* boot the core.